libs/ui-watt/src/lib/components/slider/watt-slider.component.ts

```diff
@@ -52,33 +56,86 @@ export class WattSliderComponent {
   /** Step between each value. */
   @Input() step = 1;
 
-  /** Manually define all steps. Overrules `min`, `max` and `step`. */
-  @Input() customSteps?: number[];
-
   /** The currently selected range value. */
   @Input() value: WattSliderValue = { min: this.min, max: this.max };
 
+  @ViewChild('maxRange') maxRange!: ElementRef<HTMLInputElement>;
+
+  @ViewChild('minRange') minRange!: ElementRef<HTMLInputElement>;
+
+  private _maxChangeSubscription!: Subscription;
+  private _minChangeSubscription!: Subscription;
+  private _providedStep = this.step;
   /**
    * Emits value whenever it changes.
    * @ignore
    */
   @Output() valueChange = new EventEmitter<WattSliderValue>();
 
-  /**
-   * @ignore
-   */
-  get options(): Options {
-    return {
-      floor: this.min,
-      ceil: this.max,
-      minRange: 1,
-      hideLimitLabels: true,
-      hidePointerLabels: true,
-      enforceStep: false,
-      enforceStepsArray: false,
-      step: this.step,
-      stepsArray: this.customSteps?.map((value) => ({ value })),
-    };
+  ngAfterViewInit(): void {
+    const maxRangeElement = this.maxRange.nativeElement;
+    const minRangeElement = this.minRange.nativeElement;
+    const maxChanged$ = fromEvent(maxRangeElement, 'change');
+    const minChanged$ = fromEvent(minRangeElement, 'change');
```

Comment:
issue: looks like the `change` event type comes with the downside that it only emits once when the user is done moving the slider. This means that the value shown in the time input field will not change continuously but only once. This makes it hard to know the current time value as the user moves the slider, and also to select a specific value "on the first try". Maybe going back to `input` event is better in this case because it emits values continuously.